app.js

```diff
@@ -0,0 +1,96 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const port = 3000;
+const bodyParser = require("body-parser");
+const School = require("./School.js");
+const Student = require("./Student.js");
+
+app.use(cors());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+let mySchool = new School();
+const date = new Date();
+
+
+
+const addNewClass = (req, res, next) => {
+  try{
+    let name = req.params.name;
+    let teacher = req.params.teacher;
+    if (!mySchool.classes.className) {
+      mySchool.addClass(name, teacher);
```

Comment:
You need to assign this to a variable and include that variable in your response